src/common/libsubprocess/server.c

```diff
@@ -629,10 +629,10 @@ static void server_disconnect_cb (flux_t *h,
     const char *sender;
 
     if ((sender = flux_msg_route_first (msg)))
-        server_terminate_by_uuid (s, sender);
+        subprocess_server_terminate_by_uuid (s, sender);
 }
```

Comment:
in ` libsubprocess/server: make functions local `, this change is in the wrong commit?

Reply:
oh i guess the function prototype is no longer defined b/c it's not in the header.  maybe commit message just needs tweaking, mentioning remove functions from header, adjust callers.